src/apis/doctor/doctor.service.js

```diff
@@ -368,7 +359,23 @@ class DoctorService {
         });
     }
 
-    /* for doctor activity */
+    updateDoctorStoreByDoctorId(id, type1, value1, callback) {
```

Comment:
Rename `type1` to `type` and `value1` to `value`

Reply:
we cannot make type1 to type even doctor store column is 'type'

Reply:
That's fine. It will take the key value and not the key name.